### PR TITLE
Work fe1 jwen test reducers

### PR DESCRIPTION
**Pull request by jiabaow**

### CODE REVIEW

File: fe1-web/store/reducers/__tests__/EventsReducer.test.ts [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  Hash, LaoEventType, Meeting, RollCall, RollCallStatus, Timestamp,
+} from 'model/objects';
+import {
+  addEvent,
+  clearAllEvents,
+  eventReduce, makeEventByTypeSelector, makeEventGetter,
+  makeEventsAliasMap,
+  makeEventsList, makeEventsMap, makeLastRollCallAttendeesList,
+  removeEvent,
+  updateEvent,
```
Comment (Xelowak) [inline fe1-web/store/reducers/__tests__/EventsReducer.test.ts:13]:
```suggestion
  addEvent,
  clearAllEvents,
  eventReduce,
  makeEventByTypeSelector,
  makeEventGetter,
  makeEventsAliasMap,
  makeEventsList,
  makeEventsMap,
  makeLastRollCallAttendeesList,
  removeEvent,
  updateEvent,
```

File: fe1-web/store/reducers/__tests__/LaoReducer.test.ts [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+import { AnyAction } from 'redux';
+import { Hash, Lao, Timestamp } from 'model/objects';
+import {
+  addLao,
+  clearAllLaos,
+  connectToLao,
+  disconnectFromLao,
+  laoReduce, makeCurrentLao, makeIsLaoOrganizer, makeLao, makeLaoIdsList, makeLaosList, makeLaosMap,
+  removeLao, setLaoLastRollCall,
+  updateLao,
```
Comment (Xelowak) [inline fe1-web/store/reducers/__tests__/LaoReducer.test.ts:12]:
```suggestion
  addLao,
  clearAllLaos,
  connectToLao,
  disconnectFromLao,
  laoReduce,
  makeCurrentLao,
  makeIsLaoOrganizer,
  makeLao,
  makeLaoIdsList,
  makeLaosList,
  makeLaosMap,
  removeLao,
  setLaoLastRollCall,
  updateLao,
```

File: fe1-web/store/reducers/__tests__/MessageReducer.test.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  ExtendedMessage,
+  markExtMessageAsProcessed,
+  Message,
+} from 'model/network/method/message';
+import { AddChirp } from 'model/network/method/message/data';
+import { mockPopToken } from 'model/network/method/message/__tests__/Message.test';
+import {
+  addMessages, getLaoMessagesState, getMessage, makeLaoMessagesState,
+  messageReduce,
+  processMessages,
```
Comment (Xelowak) [inline fe1-web/store/reducers/__tests__/MessageReducer.test.ts:14]:
```suggestion
  addMessages,
  getLaoMessagesState,
  getMessage,
  makeLaoMessagesState,
  messageReduce,
  processMessages,
```

File: fe1-web/model/network/method/message/__tests__/Message.test.ts [hunk 4]
```diff
@@ -29,7 +29,7 @@ const mockLao = Lao.fromState(laoState);
 
 const mockPublicKey = testKeyPair.publicKey;
 const mockPrivateKey = testKeyPair.privateKey;
-const mockPopToken = PopToken.fromState({
+export const mockPopToken = PopToken.fromState({
```
Comment (Xelowak) [inline fe1-web/model/network/method/message/__tests__/Message.test.ts:32]: If you want to make it a constant, I'd rather put it in a separate file. It feels weird to me to have a constant in this file that is exported somewhere else.

File: fe1-web/store/reducers/__tests__/SocialReducer.test.ts [hunk 5]
```diff
@@ -16,12 +16,13 @@ import {
...
-const mockLaoIdHash: Hash = Hash.fromStringArray(
-  org.toString(), new Timestamp(160000000).toString(), name,
+export const org = mockPublicKey;
+export const mockLaoName = 'MyLao';
+export const mockLaoCreationTime = new Timestamp(160000000);
+export const mockLaoIdHash: Hash = Hash.fromStringArray(
+  org.toString(), mockLaoCreationTime.toString(), mockLaoName,
 );
-const mockLaoId: string = mockLaoIdHash.toString();
+export const mockLaoId: string = mockLaoIdHash.toString();
```
Comment (Xelowak) [inline fe1-web/store/reducers/__tests__/SocialReducer.test.ts:25]: Same comment here, why should all these exported constants be in SocialReducer ?

**Review by Xelowak — DISMISSED** [review]
Nice job, thanks for all these tests ! We might be able to put ourselves in the second place in terms of coverage.
I've made some comments because I don't understand why are exported constants in these files. For example you could add a `TestUtils` file where we store all of them to avoid duplication.

**Review by pierluca — APPROVED** [review]
Approving blindly

**Pull request merged**